wemake_python_styleguide/visitors/ast/redundancy.py

```diff
@@ -53,3 +54,51 @@ def _check_for_redundant_enumerate(self, node: Union[AnyFor, ast.comprehension])
 
         if isinstance(index_receiver, ast.Name) and index_receiver.id == '_':
             self.add_violation(RedundantEnumerateViolation(index_receiver))
+
+
+@final
+class RedundantTernaryVisitor(BaseNodeVisitor):
+    """Finds useless ternary operators."""
+
+    def visit_IfExp(self, node: ast.IfExp) -> None:
+        """Finds useless ternary operators."""
+        body = ast.unparse(node.body)
+        orelse = ast.unparse(node.orelse)
+
+        # Check for specific patterns.
+        self.body_orelse_check(node, body, orelse)
+        self.check_computed_equal(node)
+
+        self.generic_visit(node)  # Visit all other node types.
+
+    def body_orelse_check(self, node, body, orelse) -> None:
```

Comment:
please, type all params here

Reply:
For this as well did you also mean the type hints?

Reply:
yes